src/app/components/team/TeamComponent.js

```diff
@@ -273,8 +266,7 @@ class TeamComponent extends Component {
         <div className="team">
           <HeaderCard>
             <ContentColumn>
-              <Message message={this.state.message} />
```

Comment:
Are error and success messages still displayed for the settings pages after this line was removed? I guess "yes" because they are now using the `FlashMessage` component?

Reply:
Yes. This state was unused actually, so this was just a simple cleanup.